dali/kernels/test/scratch_test.cc

```diff
@@ -81,31 +81,31 @@ TEST(Scratch, Estimator) {
 }
 
 TEST(Scratch, BumpAllocator) {
-  const size_t size = 1024;
-  std::aligned_storage<size, 64>::type storage;
-  BumpAllocator allocator(reinterpret_cast<char*>(&storage), size);
+  const size_t kSize = 1024;
+  alignas(64) char storage[kSize];
+  BumpAllocator allocator(storage, kSize);
```

Comment:
Here I use `char` instead of `std::byte` because they're equally good wrt aliasing and the BumpAllocator API takes `char *`.